src/grpc.ts

```diff
@@ -76,6 +76,25 @@ export class GrpcClient {
   grpc: GrpcModule;
   grpcVersion: string;
   fallback: boolean;
+  private static protoCache = new Map<string, grpc.GrpcObject>();
+
+  /**
+   * Key for proto cache map
+   */
+  private static protoCacheKey(
+    filename: string | string[],
+    options: grpcProtoLoader.Options
+  ) {
+    return JSON.stringify(filename) + ' ' + JSON.stringify(options);
```

Comment:
perhaps add a test for `undefined` and `null` options.

Reply:
Done!